client/src/main/scala/io/delta/sharing/client/DeltaSharingClient.scala

```diff
@@ -99,8 +115,32 @@ private[sharing] case class QueryTableRequest(
   maxFiles: Option[Int],
   pageToken: Option[String],
   includeRefreshToken: Option[Boolean],
-  refreshToken: Option[String]
-)
+  refreshToken: Option[String],
+  idempotencyKey: Option[String],
+  requestedColumns: Seq[String]
```

Comment:
When/who will set `requestedColumns`?

Reply:
we current don't have it supported, remove, and will add later.